Moonshot/player/States/air.gd

```diff
@@ -16,7 +16,7 @@ onready var jump_delay: Timer = $JumpDelay
 onready var controls_freeze: Timer = $ControlsFreeze
 
 export var acceleration_x := 2500.0
-
+export var jump_deceleration := 5000.0
```

Comment:
Open the player scene tree and you can edit this whilst the game is running. Make sure to save the edits to player for it to apply.

NB: This does not change the **air.gd** script, but it will probably change **player.tscn**, so make sure not to commit that as a change to the scene. If you want to suggest a different value, we will add it to this script.